docs/ReleaseNotes.md

```diff
@@ -0,0 +1,9 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+
+## Release notes
```

Comment:
Based on the readme, this is supposed to be "details about the latest release", but the initial formatting feels more like it will include the history of multiple releases. If we're trying to have per-version release notes, perhaps using the nuSpec file's releaseNotes (see [https://docs.microsoft.com/en-us/nuget/reference/nuspec#releasenotes](https://docs.microsoft.com/en-us/nuget/reference/nuspec#releasenotes)) to actually put this into the package is a cleaner option.